pyproject.toml

```diff
@@ -103,7 +103,7 @@ twilio = { version = "^8.5.0", optional = true }
 fastapi-poe = { version = "0.0.16", optional = true }
 discord = { version = "^2.3.2", optional = true }
 slack-sdk = { version = "3.21.3", optional = true }
-
+clip = {git = "https://github.com/openai/CLIP.git", optional = true}
```

Comment:
Is it possible to pin to a specific commit on main branch? It is too risky/brittle to be dependent on the head of main branch in general.